utils/node_local_test.py

```diff
@@ -1544,6 +1544,43 @@ def test_cont_copy(self):
         print(rc)
         assert rc.returncode == 0
 
+    def test_cont_clone(self):
+        """Verify that cloning a container works
+
+        This extents cont_copy, to also clone it afterwards.
+        """
+
+        # Create a temporary directory, with one file into it and copy it into
```

Comment:
@ashleypittman is there a reason that the filesystem copy code is also in the test_cont_clone function? It looks like test_cont_copy does the same thing.

Reply:
@ashleypittman ok, I see you are using a POSIX container to clone. This would work but I assumed a non-POSIX container would be used.

Reply:
@ashleypittman yeah, I suppose that is the simplest way to get a dataset into the container to clone. @daltonbohning wrote an object generator for some of the functional tests. I do not know enough about this infrastructure to know if you can access that generator for this test. This will still copy at the object level, so it is fine with me. It will just not catch certain bugs that are unique to non-POSIX datasets.

Reply:
@dsikich That "generator" wouldn't be accessible from here, because some of the functions rely on the functional test framework. And it's a little excessive for this test case, I think, since the goal here isn't necessarily to test the functionality of cont clone, but rather the memory, fault detection, etc.
For reference, the generator: https://github.com/daos-stack/daos/blob/2959f361a2a9da9c17bd1e61b92c44fd38bf27e5/src/tests/ftest/util/data_mover_test_base.py#L417

Reply:
@ashleypittman @daltonbohning ok, I think it would be "nice to have" an easier way to create non-POSIX datasets for these simple cases, since the filesystem copy already has its own test case. Not really a big deal though for these.